lib/cpl.rb

```diff
@@ -157,8 +157,13 @@ def self.all_base_commands
       desc(usage, description, hide: hide)
       long_desc(long_description)
 
+      required_options = []
       command_options.each do |option|
-        method_option(option[:name], **option[:params])
+        # We'll handle required options manually in `Config`
+        required_options.push(option[:name]) if option[:params][:required]
+
+        params = option[:params].reject { |key, _| key == :required }
```

Comment:
We remove `required` from the params list, because we handle it manually in `Config`.

Reply:
I think this is over burdening `config`.

Reply:
See: https://github.com/shakacode/heroku-to-control-plane/pull/109#discussion_r1397785197